app/src/main/java/nl/nfi/cellscanner/MainActivity.java

```diff
@@ -164,8 +164,13 @@ public void onClick(DialogInterface dialog, int which) {
      * test for the right permissions, if ok, start recording. Otherwise request permissions
      */
     public void requestStartRecording() {
-        if (PermissionSupport.hasAccessCourseLocationPermission(this)) startRecording();
-        else requestLocationPermission();
+        if (PermissionSupport.hasAccessCourseLocationPermission(this) &&
```

Comment:
deze "&&" zou een "||" moeten zijn, omdat GPS logging optioneel is

Reply:
K, dan moeten we deze als optie toevoegen aan de GUI. Op dit moment krijgen je hem er gratis bij -D

Reply:
Ik maak hier een speraat issue voor aan